statefile/load.go

```diff
@@ -8,17 +8,40 @@ import (
 	"time"
 
 	"github.com/google/uuid"
+	"github.com/turbot/steampipe/constants"
 	"github.com/turbot/steampipe/filepaths"
+	"github.com/turbot/steampipe/migrate"
 )
 
-const updateStateFileName = "update-check.json"
+const updateStateFileName = "update_check.json"
 
-// State :: the state of the installation
-type State struct {
+// LegacyState is the legacy state struct, which was used in the legacy
+// state file
+type LegacyState struct {
 	LastCheck      string `json:"lastChecked"`    // an RFC3339 encoded time stamp
 	InstallationID string `json:"installationId"` // a UUIDv4 string
 }
 
+// State :: the state of the installation
+type State struct {
+	LastCheck      string `json:"last_checked"`    // an RFC3339 encoded time stamp
+	InstallationID string `json:"installation_id"` // a UUIDv4 string
+	SchemaVersion  string `json:"schema_version"`
+}
+
+func (s State) IsValid() bool {
+	return len(s.SchemaVersion) > 0
```

Comment:
```
 return s.StructVersion > 0
```

Reply:
same for all other structs - define their own struct version,  above the struct defintion, make them int64